src/web/components/ClientSideCompletion/ClientSideCompletion.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { faker } from '@faker-js/faker';
```

Comment:
ClientSideCompletion is kind of in a random place in Storybook - maybe move it under the CSTG section?